lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -531,11 +531,14 @@ module.exports = class MethodMappings {
           return `/element/${id}/location_in_view`;
         },
 
-        async isElementDisplayed(webElementOrId) {
+        isElementDisplayed(webElementOrId) {
+          if (this.transport.api.isAppiumClient()) {
```

Comment:
We should not use api methods here. We can move isAppium client to util and api method can use take use of that util function

Reply:
Following up on the above response, this method cannot be moved to utils in its current form since for handling the BrowserStack case, we'd need the `capabilities` returned by BrowserStack after a session is established with it, and those `capabilities` are only made available in `client.js` as `this.capabilities`.